torch/functional.py

```diff
@@ -981,6 +984,104 @@ def cdist(x1, x2, p=2., compute_mode='use_mm_for_euclid_dist_if_necessary'):
     else:
         raise ValueError("{} is not a valid value for compute_mode".format(compute_mode))
 
+def atleast_1d(*tensors):
+    r"""
+    atleast_1d(input) -> Tensor
```

Comment:
`atleast_1d(input) -> Tensor or list of Tensors`?

Reply:
Will remove this line as we already have https://github.com/pytorch/pytorch/blob/270093825b16cdf4445998c8c83b8db6edf2ca61/torch/functional.py#L993-L994